src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -8,11 +8,10 @@
 from __future__ import division, print_function, absolute_import
 
 import logging
-
+from copy import deepcopy
 from rucio.client import Client
 from rucio.common.exception import AccountNotFound, DataIdentifierNotFound,\
```

Comment:
Prettier way to do this is use () around the exception names. Preferred now too.

Reply:
will update it